code/modules/surgery/organs/organ_internal.dm

```diff
@@ -266,36 +266,6 @@
 				H.adjustFireLoss(-cursed_heart.heal_burn)
 				H.adjustOxyLoss(-cursed_heart.heal_oxy)
 
-/obj/item/organ/internal/lungs
-	name = "lungs"
-	icon_state = "lungs"
-	gender = PLURAL
-	organ_tag = "lungs"
-	parent_organ = "chest"
-	slot = "lungs"
-	vital = 1
-
-//Insert something neat here.
-///obj/item/organ/internal/lungs/remove(mob/living/carbon/M, special = 0)
-//	owner.losebreath += 10
-	//insert oxy damage extream here.
-//	. = ..()
-
-
-/obj/item/organ/internal/lungs/on_life()
-	..()
-	if(germ_level > INFECTION_LEVEL_ONE)
-		if(prob(5))
-			owner.emote("cough")		//respitory tract infection
-
-	if(is_bruised())
```

Comment:
Cant this just be moved to check_breath(), and checking for organ damage if it comes down to it? Like 
```
if( damage >= 10 && damage < 20 )
  owner.custom_emote( 1, "coughs up blood!" )
 owner.bleed( 1 )
else if( damage > 20 )
   owner.custom_emote( 1, "gasps for air!" )
   owner.AdjustLoseBreath( damage )
```
Notice I'm scaling losebreath() with the amount of lung damage, or at least that was the idea

Im very iffy at throwing away punctured lungs just because some 10 lines of code look ugly. They have game implications different from standing in a low 02 environment, in that if you step into space without internals you will need surgery or a chem to pay for it or you will have long term consequences, opposed to lel i was in space with perfluo but its fine now im inside my lungs are made of steal nothing happened kk